application.py

```diff
@@ -12,10 +12,8 @@
 # Configure logging to emit to stdout
 logging.basicConfig(
     level=logging.INFO,
-    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
-    handlers=[
-        logging.StreamHandler(sys.stdout)
-    ]
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    handlers=[logging.StreamHandler(sys.stdout)],
```

Comment:
This is properly (i.e. `black`) formatted. I've seen diffs like this in a few PRs now that are indicative of editor config bounce. Your version is correct, but I've created a ticket XREF #654 https://github.com/ua-snap/data-api/issues/654